LoopKit/Alert.swift

```diff
@@ -35,15 +35,21 @@ public struct PersistedAlert: Equatable {
     }
 }
 
-/// Protocol for looking up alerts persisted in storage
+/// Protocol for recording and looking up alerts persisted in storage
 public protocol PersistedAlertStore {
+    /// Determine if an alert is already recorded for a given `Alert.Identifier`.
```

Comment:
Any alert, regardless of retraction?  Also, this is already `PersistedAlertStore`, so no need IMO to repeat `Persisted`.  Consider naming `doesIssuedAlertExist` instead (or `doesUnretractedAlertExist` or whatever condition this API specifies).  Also consider updating the comment.

Reply:
This is intended to find an alert irrelevant of its state ([un]retracted | [un]acknowledged). Just that it exists in the store.